test-xml-parsing/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>sonar-analyzer-commons-parent</artifactId>
+    <groupId>org.sonarsource.analyzer-commons</groupId>
+    <version>1.11-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>test-sonar-xml-parsing</artifactId>
+  <name>SonarSource Analyzers XML Parsing Test Commons</name>
+  <description>Logic useful to test XML parsing and XML-related rules</description>
+
+  <dependencies>
+    <!-- PROVIDED -->
+    <dependency>
+      <groupId>org.sonarsource.sonarqube</groupId>
+      <artifactId>sonar-plugin-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.sonarsource.analyzer-commons</groupId>
+      <artifactId>sonar-xml-parsing</artifactId>
+      <scope>provided</scope>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.sonarsource.analyzer-commons</groupId>
+      <artifactId>sonar-analyzer-test-commons</artifactId>
+      <scope>provided</scope>
```

Comment:
I'm not sure why we use `provided` (rather than the default `compile`) here and a few lines above for `sonar-xml-parsing`.